CHANGELOG.md

```diff
@@ -104,7 +109,6 @@
 - Release Process (Apps & Services)
   - information message "no {object} available" getting displayed while content is available
 - App Release Process
-  - fixed automatic upscrolling in form pages
```

Comment:
did you intend to change the changelog from a already released version?

Reply:
@evelyn It was added twice(line 107 and line113), so removed duplicate log

Reply:
> @evelyn It was added twice(line 107 and line113), so removed duplicate log

@evegufy  I have reverted the changelog from previous release